### ISSUE TITLE
Disabled Webkit wafer map tests due to OffscreenCanvas

### ISSUE THREAD

**Issue by m-akinc** [ni/nimble#2169]
# 🧹 Tech Debt

The wafer map uses the `OffscreenCanvas` feature that [was added to Safari in version 17](https://caniuse.com/offscreencanvas), but is [explicitly disabled in the WebKit build (see collapsed comment)](https://github.com/microsoft/playwright/issues/31004#issuecomment-2131219442) for Windows/Linux platforms. The WebKit commit for the `OffscreenCanvas` feature [confirms this (see LayoutTests/platform/win/TestExpectations)](https://github.com/WebKit/WebKit/commit/b32d7b9f1487e0c5bcd836a8dda7cc5fc44485a6#diff-14e0f5ed6cdc77e29f31c08246c37593c9a855541a6d4d5bc654d7032c67dcae).

Therefore, we had to disable any wafer map tests that exercise `OffscreenCanvas` code in WebKit:

- `nimble-components/build/generate-workers/source/tests/matrix-renderer.spec.ts` (all tests)
- `nimble-components/src/wafer-map/tests/create-matrix-renderer.spec.ts` (all tests)
- `nimble-components/src/wafer-map/tests/hover-handler.spec.ts` (all tests)
- `nimble-components/src/wafer-map/tests/wafer-map.spec.ts`:
   - "will call setupWafer and drawWafer after supported diesTable change"
   - "worker renderer action" `describe` block
   - "can still zoom after canvas height was changed"
   - "can still zoom after canvas width was changed"

**Comment by rajsite** [comment]
If I'm reading https://github.com/microsoft/playwright/issues/31004 right should we should expect OffscreenCanvas support in the next playwright release?

**Comment by m-akinc** [comment]
Hopefully, but it's hard to tell for sure. You think we should leave the tests enabled until next quarter's manual maintenance pass?

**Comment by rajsite** [comment]
Makes sense they were disabled in #2171 so those who do try and test in WebKit don't hit unexpected issues. Added a note to teh recurring playwright task to double check this issue and see if the tests can be re-enabled.

**Comment by mxschmitt** [comment]
Might be relevant: It should be also fixed in our [Canary releases](https://playwright.dev/docs/canary-releases).
